dbt_project.yml

```diff
@@ -24,6 +24,9 @@ models:
     ticket_history:
       +schema: zendesk_intermediate
       +materialized: ephemeral
+    unstructured:
+      +schema: zendesk_unstructured
+      +materialized: table
```

Comment:
Let's sync on this default schema

Reply:
From our discussion, leaving this as-is so it is separated.